exec/java-exec/src/test/java/org/apache/drill/exec/rpc/data/TestBitRpc.java

```diff
@@ -94,6 +94,39 @@ public void testConnectionBackpressure() throws Exception {
     Thread.sleep(5000);
   }
 
+  @Test
+  public void testConnectionBackpressureWithDynamicCredit() throws Exception {
+    final WorkerBee bee = mock(WorkerBee.class);
```

Comment:
There is a concept of effective final in java so it is not necessary to add final everywhere except of the cases you really won't to make a statement.